go.mod

```diff
@@ -135,7 +136,6 @@ require (
 	github.com/dop251/goja v0.0.0-20230806174421-c933cf95e127 // indirect
 	github.com/elastic/gosigar v0.14.2 // indirect
 	github.com/ethereum/c-kzg-4844 v0.4.0 // indirect
-	github.com/ferranbt/fastssz v0.0.0-20210120143747-11b9eff30ea9 // indirect
```

Comment:
Why do we have this change here?

Reply:
I have no idea to be honest. That's what go mod tidy gave me. Any clue?

Reply:
`broadcaster_test.go` imports and uses ssz. I suspect this causes it.

Reply:
I was using wrong one. Replaced it with `github.com/prysmaticlabs/fastssz` and resolved.